code/modules/power/apc.dm

```diff
@@ -184,9 +184,6 @@
 /obj/machinery/power/apc/highcap/ten_k
```

Comment:
the highcap type doesnt exist, remove this subtype and make it the highcap type

Reply:
The highcap type itself doesn’t exist, but there’s still also the 5k version